docs/src/main/asciidoc/security-jwt-build.adoc

```diff
@@ -30,14 +32,15 @@ SmallRye JWT Build provides an API for securing JWT claims using all of these op
 </dependency>
 ----
 
+Alternatively, for a Gradle-based project, use the following configuration:
+
 [source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
 .build.gradle
 ----
 implementation("io.quarkus:quarkus-smallrye-jwt-build")
 ----
 
-Note you can use SmallRye JWT Build API without having to create MicroProfile JWT endpoints supported by `quarkus-smallrye-jwt`.
-It can also be excluded from `quarkus-smallrye-jwt` if MP JWT endpoints do not need to generate JWT tokens.
+You can use the SmallRye JWT Build API independently, without needing to create MicroProfile JWT endpoints supported by the `quarkus-smallrye-jwt` extension. Additionally, the API can be excluded from `quarkus-smallrye-jwt` if there is no requirement to generate JWT tokens for MicroProfile JWT endpoints.
```

Comment:
```suggestion
You can use the SmallRye JWT Build API independently, without needing to create MicroProfile JWT endpoints supported by the `quarkus-smallrye-jwt` extension.
```
Not related to the PR, but this is an old suggestion which might work if the optimization is needed but more likely will cause side-effects, with the latest devmode support work, and in native, so better remove it